README.markdown

```diff
@@ -413,6 +415,32 @@ attendeeList.sort { a, b in
 }
 ```
 
+To condense the closure even further, remove any `parentheses` and unless the closure returns something other than `Void`, remove `arrows(->)` as well.
```

Comment:
use the following to encase the return arrow operator:

"...remove the return arrow (`->`)..."